.env

```diff
@@ -0,0 +1,8 @@
+# Environment variables declared in this file are automatically made available to Prisma.
```

Comment:
Instead of committing a .env file, could you commit a .env.example file instead, and add .env to the ignore? Someone might accidentally commit secrets to this file. To get the project up and running they can just copy the .env.example and rename it to .env